Tests/SonarScanner.MSBuild.Tasks.UnitTests/MoveDirectoryTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿/*
+ * SonarScanner for MSBuild
+ * Copyright (C) 2016-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.IO;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarScanner.MSBuild.Tasks;
+using SonarScanner.MSBuild.Tasks.UnitTests;
+using TestUtilities;
+
+namespace SonarScanner.Integration.Tasks.UnitTests
+{
+    [TestClass]
+    public class MoveDirectoryTests
+    {
+        private MoveDirectory sut;
```

Comment:
I don't like this shared state in this case. Orchestration from one test can affect results of the other one if we're not careful enought.